pkgs/applications/science/medicine/dcmtk/0001-Fix-cmake.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/CMake/dcmtk.pc.in b/CMake/dcmtk.pc.in
+index 13c79c0..f27acaa 100644
+--- a/CMake/dcmtk.pc.in
++++ b/CMake/dcmtk.pc.in
+@@ -1,6 +1,6 @@
+  prefix="@CMAKE_INSTALL_PREFIX@"
+  exec_prefix="${prefix}"
+- libdir="${prefix}/@CMAKE_INSTALL_LIBDIR@"
++ libdir="@CMAKE_INSTALL_LIBDIR@"
```

Comment:
If you make it CMAKE_INSTALL_FULL_LIBDIR you can submit the patch upstream.

Reply:
Done https://github.com/DCMTK/dcmtk/pull/75

Reply:
Thanks! I made some comments over there.